Changelog.md

```diff
@@ -5,12 +5,12 @@
 * Fix issue #2640 (leaked `ClosureTable` entry when awaiting futures fails).
 
 * Vastly improved garbage collection scheduling: previously Motoko runtime would do GC
-  after every upgrade message. We now schedule a GC when
+  after every update message. We now schedule a GC when
 
   1. Heap grows more than 50% and 10 MiB since the last GC, or
   2. Heap size is more than 3 GiB
 
-  (1) is to make sure we don't do GC on tiny heaps or after only small amounts of allocation. (2) is to make sure that
+  (i) is to make sure we don't do GC on tiny heaps or after only small amounts of allocation. (ii) is to make sure that
```

Comment:
These refer to the (1) and (2) above, why confuse the reader by spelling them Roman here?

Reply:
Since the Roman numerals come from GitHub's braindead CSS, the HTML should be fine elsewhere. So I went back to the original.